shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/MetaData.hs

```diff
@@ -31,6 +31,8 @@ import           GHC.Generics (Generic)
 import           Shelley.Spec.Ledger.Serialization (mapFromCBOR, mapToCBOR)
 
 -- | A generic metadatum type.
+--
+-- TODO make strict
```

Comment:
I'll come back to address this. Either by using a proper library of strict data types or by forcing them when creating them.